Formula/bash-preexec.rb

```diff
@@ -1,5 +1,5 @@
 class BashPreexec < Formula
-  desc "preexec and precmd functions for Bash just like Zsh."
```

Comment:
Ah yes, the `"`. Maybe half past seven in the morning isn't the best time to be thinking about regexes. 😴